package.json

```diff
@@ -77,7 +77,7 @@
     "react-dom": "^15.4.2",
     "react-router": "^3.0.2",
     "sauce-connect-launcher": "^1.2.0",
-    "selenium-server-standalone-jar": "3.1.0",
+    "selenium-server-standalone-jar": "3.2.0",
```

Comment:
Since this package is used in `/src` it should be moved to a dependency in the terra-toolkit package. This will ensure that this package is installed with terra-toolkit.